app/src/main/java/com/onboarding/wheatherapp/data/services/WeatherService.kt

```diff
@@ -26,5 +29,6 @@ class WeatherService {
     companion object {
         private const val APPID = "90c4a6b89d1036c5fb537a1a3582e818"
```

Comment:
put this APPID in a variable on gradle pls

Reply:
Solved!